src/Allure.XUnit.Examples/ExampleParameterisedTests.cs

```diff
@@ -56,5 +68,15 @@ public void TestTheory(int a, int b)
         {
             Assert.Equal(a, b);
         }
+
+        [AllureXunitTheory]
+        [AllureParentSuite("AllTests")]
+        [AllureSuite("Test AllureXunitTheory")]
+        [AllureSubSuite("Test MemberData Class")]
+        [MemberData(nameof(TestClassData))]
```

Comment:
Your changes are broking InlineDataAttribute and ClassDataAttribute working. Tests set are displaying as one test.This problem is actual for MemberDataAttribute too.

Reply:
Very strange behavior. 
If Test class is serializable XunitTestCase inheritor works fine.
If Test class is non serializable XunitTheoryTestCase inheritor works fine.